test/validate.js

```diff
@@ -18,7 +18,7 @@ mkdirp.sync(dir)
 
 const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
 
-const sbot = SecretStack({ appKey: caps.shs }).use(require('../')).call(null, {
+const sbot = SecretStack({ appKey: caps.shs }).use(require('../')).use(require('../compat/post')).call(null, {
```

Comment:
why is this one in here?

Reply:
because `db.post.value` is used further down in these tests